ingestion/src/metadata/cli/docker.py

```diff
@@ -47,7 +55,31 @@
 }
 
 
+def docker_volume():
+
+    # create a main directory
+    maindir = "docker-volume"
+    if not os.path.exists(maindir):
+        os.mkdir(maindir)
+        db = "db-data"
+        dag_airflow = "ingestion-volume-dag-airflow"
+        dags = "ingestion-volume-dags"
+        tmp = "secrets"
+        om_server = "om-server"
+        final_db_path = os.path.join(maindir, db)
+        final_dag_airflow_path = os.path.join(maindir, dag_airflow)
+        final_dags_path = os.path.join(maindir, dags)
+        final_tmp_path = os.path.join(maindir, tmp)
+        final_om_server = os.path.join(maindir, om_server)
+        os.makedirs(final_db_path, exist_ok=True)
+        os.makedirs(final_dag_airflow_path, exist_ok=True)
+        os.makedirs(final_dags_path, exist_ok=True)
+        os.makedirs(final_tmp_path, exist_ok=True)
+        os.makedirs(final_om_server, exist_ok=True)
```

Comment:
we can clean this up a little better, let me know if you need any help around here..

we can loop it up, and make directories in the same loop